gate-web/src/main/groovy/com/netflix/spinnaker/gate/security/AuthConfig.groovy

```diff
@@ -62,26 +55,11 @@ class AuthConfig {
   @Autowired
   FiatPermissionEvaluator permissionEvaluator
 
-  @Bean
-  @ConditionalOnMissingBean(UserRolesProvider)
-  UserRolesProvider defaultUserRolesProvider() {
-    return new UserRolesProvider() {
-      @Override
-      Map<String, Collection<String>> multiLoadRoles(Collection<String> userEmails) {
-        return [:]
-      }
-
-      @Override
-      Collection<String> loadRoles(String userEmail) {
-        return []
-      }
-    }
-  }
-
   void configure(HttpSecurity http) throws Exception {
     // @formatter:off
     SecurityBuilder result = http
       .authorizeRequests()
+        .antMatchers('/**/favicon.ico').permitAll()
```

Comment:
this was necessary to avoid invalidating the session in FiatSecurityFilter during the auth handshake (at least for SAML)

In the boot 1.2.8 based version this (and a couple other paths /js/**, /images/**) were getting added as default ignored paths, something has changed in the way that stuff bootstraps now so that config doesn't automatically apply.

The only one that mattered was  flavor flavicon :clock1: because we aren't serving any of the other asset types